pixl_core/src/core/db/models.py

```diff
@@ -52,6 +52,7 @@ class Image(Base):
     accession_number: Mapped[str]
     study_date: Mapped[Date] = mapped_column(Date())
     mrn: Mapped[str]
+    study_uid: Mapped[str]
```

Comment:
You'll also need to make an alembic migration for this change

Reply:
Oh this also should be optional, or we populate with a dummy value when we create the column